pyproject.toml

```diff
@@ -24,9 +29,13 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8.1,<3.13"
+python = ">=3.9,<3.14"
 requests = ">=2.26.0"
-pendulum = ">=2.1.2"
+# TODO: pin to tag on dlt-pendulum repo
+dlt-pendulum = [
```

Comment:
NOTE: this still requires cargo at this point

Reply:
@rudolfix I have another question about this: should we not have a different package name for our pendulum? The way it is currently set up, dlt-pendulum is always used, also for python versions before 3.13, this means there will be package name collissions if users also have the regular pendulum in their dependencies already in a project. I'm not sure what happens in this case.

Also you mentioned that you made some changes they will not like and there is a change in the code where you removed the casting of pendulum.now() to a string, are the libraries now incompatible?

Reply:
We are now using a properly build dlt-pendulum for python 3.13, so that part is solved. The tests will verify wether this string removal still works or not.